pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -126,3 +127,18 @@ func (h *DHCPv6Handler) ServeDHCPv6(conn net.PacketConn, peer net.Addr, m dhcpv6
 		log.Log.V(4).Errorf("DHCPv6 cannot reply to client: %v", err)
 	}
 }
+
+func prepareDHCPModifiers(
```

Comment:
How about using the build pattern here?
Each returning a single "modifier". It should help with unit testing and it will give naming to each modifier nicely.

Reply:
I think it is very simple the way it is. dhcpv6 is not going to be used/built by multiple developers that will have different combinations of modifiers.
We have just one single combination of modifiers, I believe the build pattern doesn't have added value here.